.php-cs-fixer.php

```diff
@@ -4,7 +4,7 @@
 	[
 		__DIR__ . '/app/',
 		__DIR__ . '/database/',
-		__DIR__ . '/lang/',
+		// __DIR__ . '/lang/', // We no longer format lang files as they are from weblate and weblate does not support our formatting.
```

Comment:
Is this worth leaving as a comment or should it be removed entirely?

Reply:
I would like to keep it because at a later stage I would be happy to have this formatted with a secondary workflow: 
if files is not formatted, automatically create a PR to format it so that formatting can be enforced in such case (or figure out how to fix the formatting on weblate side :/

Reply:
I am also not closed to removing the lang from formatting completely.

Reply:
Fixed: [60488cb](https://github.com/LycheeOrg/Lychee/pull/3322/commits/60488cb8a0bd89dfa0e49c42e257da657975948c)